examples/unitTestExample/README.md

```diff
@@ -0,0 +1,27 @@
+# Running The Tests
+Run them in Android studio or run them via gradle: 
+
+Instrumentation Tests
+`./gradlew connectedCheck`
+
+Unit Tests
+`./gradlew test` 
+
+# Troubleshooting
+
+Depending on what version of Java you have installed you may encounter the following errors: 
+
+`initializationError(com.your.ClassHere): Bad <init> method call from inside of a branch`
+  
+or
+ 
+`Exception in thread "main" java.lang.annotation.AnnotationFormatError: Invalid default: public abstract java.lang.Class org.robolectric.annotation.Config.application()` 
+
+These issues manifest themselves because of changes to the Java bytecode verifier. 
+A workaround is to set the `-noverify` flag on the the JVM. 
+
+
+Typically, the tests can be run via the command line without fail, however, to get them to run in Android Studio you will need to perform the following steps in your run configurations in Android Studio:
```

Comment:
It's an idiom in English (http://idioms.thefreedictionary.com/without+fail). However, I can see how it might be confusing. Do you think it's worth changing? Either way, its not a big deal to me. :)